examples/typescript-esm/codegen.cjs

```diff
@@ -0,0 +1,16 @@
+// @ts-check
+
+/** @type {import("@graphql-codegen/cli").CodegenConfig} */
+const config = {
+  schema: 'https://swapi-graphql.netlify.app/.netlify/functions/index',
+  documents: ['src/**/*.ts'],
+  emitLegacyCommonJSImports: false,
+  generates: {
+    './src/gql/': {
+      plugins: [],
+      preset: 'client-preset',
+    },
+  },
+};
+
+module.exports = config;
```

Comment:
Wanted to use `.ts`, but couldnt due to cosmicconfig using `require` instead of `import`.
Tried using `.mts`, but cosmicconfig complained about an unknown file extension 😬